src/gwt/src/org/rstudio/studio/client/workbench/views/output/lint/LintManager.java

```diff
@@ -222,7 +226,7 @@ private void performLintServerRequest(final LintContext context)
 
       if (userPrefs_.showDiagnosticsCpp().getValue() && (target_.getTextFileType().isCpp() || target_.getTextFileType().isC()))
          performCppLintServerRequest(context);
-      else if (userPrefs_.showDiagnosticsR().getValue() && target_.getTextFileType().isR())
+      else if (userPrefs_.showDiagnosticsR().getValue() && (target_.getTextFileType().isR() || target_.getTextFileType().isRmd()))
```

Comment:
Should this file type list match the one above? (i.e. R, Rmd, Rnw, Rpres)

Reply:
In this case, no -- code diagnostics are only currently enabled for R and R Markdown documents specifically; for the other multimode R documents I think we just want spellchecking to run (if enabled).